src/services/azureBlobStorageService.ts

```diff
@@ -149,4 +155,12 @@ export class AzureBlobStorageService extends BaseService {
       blobName,
     );
   }
+
+  private async getToken(): Promise<string> {
+    const authResponse = await AzureLoginService.login({
+      tokenAudience: "https://storage.azure.com/"
```

Comment:
if this is only used here, I'm good, otherwise let's promote to a const somewhere.